diff --git a/living_room_leds/living_room_leds.ino b/living_room_leds/living_room_leds.ino
index 3e8c998..fa1d168 100644
--- a/living_room_leds/living_room_leds.ino
+++ b/living_room_leds/living_room_leds.ino
@@ -21,19 +21,19 @@ Adafruit_NeoPixel strip = Adafruit_NeoPixel(PIXEL_COUNT, PIXEL_PIN, NEO_GRB + NE
 int currcolor = 0;
 
 #define BTNTHRES 110
-#define LEFTCORNER 91
-#define CENTERPIX 195
-#define NMODES 6
+#define LEFTCORNER 0
+#define CENTERPIX 150
+#define NMODES 3
 #define NHIST 30
 // Set to -1 to turn replay off
 int replay_ind = -1;
 int hist[NHIST];
-int mode = 5;
+int mode = 0;
 int lastxdir = 0;
 int lastydir = 0;
 int record_ind;
 int record_time;
-int frame = 0;
+unsigned int frame = 0;
 
 
 void setup() {
@@ -175,59 +175,11 @@ void loop() {
     }
   } else if (mode == 2) { // strobe
     for (int i = LEFTCORNER; i < PIXEL_COUNT; i += 1) {
-      if (frame % 4+y/64 >= 2)
+      if (frame % 4+y/64 >= 2 && y > -100)
         strip.setPixelColor(i, 0x666666);
       else
         strip.setPixelColor(i, 0);
     }
-  } else if (mode == 3) {
-    static long n = 0;
-    for (int i = 0; i < sizeof(long)*8; i++) {
-      strip.setPixelColor(i, (n >> i) & 1 ? 0xff0000 : 0);
-    }
-    n++;
-  } else if (mode == 4) {
-    for (int i = LEFTCORNER; i < PIXEL_COUNT; i += 1) {
-        strip.setPixelColor(i, Wheel(y+128, 128));
-    }
-  } else if (mode == 5) {
-    const int rule = 30;
-    const uint32_t automataColor = 0x000060;
-    if (frame == 0) {
-      // Initial condition
-      for (int i = 0; i < PIXEL_COUNT; i++) {
-        strip.setPixelColor(i, 0);
-      }
-      strip.setPixelColor(PIXEL_COUNT-1, automataColor);
-    }
-    if (frame % 4 == 0) {
-      int last = 0;
-      int curr = 0;
-      int next = 0;
-      for (int i = 0; i < PIXEL_COUNT; i++) {
-        last = !!strip.getPixelColor(i);
-        curr = (i<PIXEL_COUNT) ? !!strip.getPixelColor(i+1) : 0;
-        next = (i+1<PIXEL_COUNT) ? !!strip.getPixelColor(i+2) : 0;
-        
-        int id = next | (curr<<1) | (last<<2);
-        int c = !!(rule & (1<<id));
-        
-        /*if(frame == 1*20 && i < 3) {
-          Serial.println("Inner Loop");
-          Serial.println(i);
-          Serial.println(last);
-          Serial.println(curr);
-          Serial.println(next);
-          Serial.println(id);
-          Serial.println(c);
-          Serial.print(c);
-          Serial.print(" ");
-        }
-        Serial.println("");*/
-        
-        strip.setPixelColor(i, c ? automataColor : 0);
-      }
-    }
   }
   
   strip.show();
